.github/workflows/run-integration-test.yml

```diff
@@ -59,5 +57,15 @@ jobs:
           echo "Found from previous job URL=$URL"
           echo "Found from previous job SECOND_URL=$SECOND_URL"
         env:
-          URL: ${{ fromJson(steps.store-variables.outputs.variables).URL }}
-          SECOND_URL: ${{ fromJson(steps.store-variables.outputs.variables).SECOND_URL }}
\ No newline at end of file
+          URL: ${{ env.URL }}
+          SECOND_URL: ${{ env.SECOND_URL }}
+
+  should-not-fail-on-retrieving:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./
+        with:
+          variables: |
+            RANDOM_VAR
+          failIfNotFound: false # default is false
```

Comment:
Is it a good idea to add a test about failing to read a variable?

Like, run the test, expect a failure, and consider the failure as a success? Or is it not possible because it'll crash the CI?

Reply:
I was thinking about it but if a step crash, it will say that the workflow is not working, and it's not really what we want... I already had a look to override a step status, but it's not possible. I did not focus on tests, for now, anyway.

Reply:
Okay, too bad. Seems to be a much wanted feature.
https://github.com/actions/toolkit/issues/399#issuecomment-615107794